components/mailersend/mailersend.app.mjs

```diff
@@ -1,11 +1,186 @@
+import {
+  EmailWebhook,
+  MailerSend,
+  EmailParams,
+  Sender,
+  Recipient,
+} from "mailersend";
+
 export default {
   type: "app",
   app: "mailersend",
-  propDefinitions: {},
+  propDefinitions: {
+    domainId: {
+      type: "string",
+      label: "Domain ID",
+      description: "The id of a verified domain.",
+      async options({ prevContext }) {
+        const page = (prevContext?.nextPage || 0);
```

Comment:
I could not get what brackets do here. Is it on purpose? If not can we remove them?

Reply:
Sure it can be removed